src/main/java/blackjack/domain/Dealer.java

```diff
@@ -0,0 +1,27 @@
+package blackjack.domain;
+
+public class Dealer extends Participant {
+
+    public Dealer(String name) {
+        super(name);
+    }
+
+    @Override
+    public void drawCardMultiple(Deck deck, int number) {
+        cards.addCards(deck.drawMultiple(number));
+    }
+
+    @Override
+    public void drawCardContinue(Deck deck) {
```

Comment:
```suggestion
    public boolean drawCardContinue(Deck deck) {
```
이렇게 수정해 컨트롤러에 메시지를 던지는 방식으로 수정해보는 것은 어떨까요?
그리고 21점이 넘으면 카드를 받을 수 없는 조건이 빠져있습니다.